rca/shortcourses/migrations/0031_auto_20220203_1140.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.11 on 2022-02-03 11:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("shortcourses", "0030_add_contact_fields"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="shortcoursepage", name="access_planit_course_id",
+        ),
+        migrations.AlterField(
+            model_name="shortcoursemanualdate",
+            name="booking_link",
+            field=models.URLField(blank=True),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursemanualdate",
+            name="cost",
+            field=models.PositiveIntegerField(null=True),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursemanualdate",
+            name="end_date",
+            field=models.DateField(null=True),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursemanualdate",
+            name="start_date",
+            field=models.DateField(null=True),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursepage",
+            name="application_form_url",
+            field=models.URLField(
+                blank=True,
+                help_text="The URL of the application form. This will be a direct link and won't open the modal",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursepage",
+            name="manual_registration_url",
+            field=models.URLField(
+                blank=True, help_text="The register interest link shown in the modal"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="shortcoursepage",
+            name="show_register_link",
+            field=models.BooleanField(
+                default=1,
+                help_text="If selected, an automatic 'Register your interest' link will be visible in the key details section",
+            ),
+        ),
+    ]
```

Comment:
**Suggestion**
Name this migration file so we can refer to it easier if we ever have to re-implement this